services/core-web/common/reducers/partiesReducer.js

```diff
@@ -84,5 +89,8 @@ export const getLastCreatedParty = (state) => state[PARTIES].lastCreatedParty;
 export const getInspectors = (state) => state[PARTIES].inspectors;
 export const getInspectorsList = (state) =>
   createDropDownList(state[PARTIES].inspectors, "name", "party_guid");
+export const getProjectLeads = (state) => state[PARTIES].project_leads;
+export const getProjectLeadsList = (state) =>
+  createDropDownList(state[PARTIES].project_leads, "name", "party_guid");
 
```

Comment:
'getProjectLeadsList' should be in the selector, but I see that Inspectors is also incorrectly transformed here. We should be importing all redux data into components via selectors, not reducers.